doc/manual/rule_uid_schema.md

```diff
@@ -0,0 +1,189 @@
+<!---
+This Source Code Form is subject to the terms of the Mozilla
+Public License, v. 2.0. If a copy of the MPL was not distributed
+with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Rule UID Schema
+
+## Introduction
+
+The document describes the composition of the unique identifier (UID) to be
+associated with quality checker rules.
+
+The document describes composition of rule UIDs, declares some of the concepts
+to be used for ASAM rules, and provides an example for an UID for a rule of the
+ASAM OpenDRIVE standard.
+
+## Rule UID Concepts
+
+The UID is a string which encapsulate a sequence of **concepts** that allow to
+identify immediately a rule across the different domains. The concept are
+ordered and separated via the separation character **`:`**.
+
+The concepts for the rule UID are:
+
+ * **Emanating Entity**: a domain name for the entity (organization or company)
+   that declares the rule UID
+ * **Standard**: a short string that represents the standard or the
+   domain to which the rule is applied
+ * **Definition Setting**: the version of the standard or the domain to which
+   the rule appears or is applied **for the first time**
```

Comment:
Has it already been discussed how to proceed if a rule no longer applies to a standard from a certain version onwards? Or whether it only applies to certain versions, e.g. "1.0.0, 1.7.0" (I'm not sure if this is happening)? This information is probably not necessary in the rule definition, but could be helpful if we want to check whether a certain rule applies to a certain file.

Reply:
Good catch! Yes it has been discussed. It is not included in the rule uid, but it is included in the rule metadata. It is difficult, in general, at the moment of uid creation to know when a rule will not be valid anymore (uid are immutable). Thus at the moment we are only setting the first time a rule appeared.

Cfr. #5 Pain Points and Weaknesses point 2

Reply:
Alright :) with the ability to specify the applicable version range, it's fine with me.